### ISSUE TITLE
Test suite currently broken

### ISSUE THREAD

**Issue by conorsch** [freedomofpress/securedrop-workstation#42]
After following the README and running `make all` without errors, I'm unable to run the test suite:

```
$ make test
tests (unittest.loader.LoadTestsFailure) ... ERROR

======================================================================
ERROR: tests (unittest.loader.LoadTestsFailure)
----------------------------------------------------------------------
AttributeError: 'module' object has no attribute 'vms_exist'

----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
Makefile:73: recipe for target 'test' failed
make: *** [test] Error 1
```

Using the latest master branch, and made sure to run `make clone` before `make all`.

**Comment by conorsch** [comment]
Looks like the test suite broke in the Qubes 3.2 -> 4.0-rc* transition. We'll need to port the tests to use the new libraries.

**Comment by joshuathayer** [comment]
This should be fixed by #47

**Comment by conorsch** [comment]
Resolved by #47. Some follow-up work is forthcoming, related to #50.
